test_email_validation.py

```diff
@@ -8,7 +8,21 @@ class EmailValidationTest(unittest.TestCase):
     def test_validate_email_pass(self):
         """Check whether validate_email function pass."""
         self.assertTrue(validate_email('123@123.com'))
+        self.assertTrue(validate_email('abc@abc.com'))
+        self.assertTrue(validate_email('abc123@abc123.com'))
+        self.assertTrue(validate_email('a.bc_123@abc123.com'))
+
 
     def test_validate_email_false(self):
         """Check whether validate_email function false."""
         self.assertFalse(validate_email('АБВ123@123.com'))
+        self.assertTrue(validate_email('123@абв.com'))
```

Comment:
This email *is valid* (taking into account IDN), the thing is whether you want function to accept ASCII-encoded Cyrillic or unicode.